components/x-topic-search/src/lib/get-suggestions.js

```diff
@@ -3,39 +3,47 @@ const addQueryParamToUrl = (name, value, url, append = true) => {
 	return append === true ? `${url}&${queryParam}` : `${url}?${queryParam}`;
 };
 
-const suggest = function (suggestions, followedTopics, searchTerm) {
+const separateFollowedAndNot = (suggestions, followedTopics) => {
+	const matchingFollowedTopics = [];
+	const matchingUnfollowedTopics = [];
+
+	if (!followedTopic.length) {
```

Comment:
I don't think this is necessary. If `followedTopics` is an empty array, then the `.forEach` below will effectively do nothing, and then the function can just `return { matchingFollowedTopics, matchingUnfollowedTopics }` (which will be two empty arrays).